src/components/ListHeader.js

```diff
@@ -11,12 +9,10 @@ export const ListHeader = ({ queryVariables, data, updateListData }) => {
   const dataProviders =
     data &&
     data.dataProviders &&
-    _sortBy(_uniq(data.dataProviders.map((item) => item.dataProvider.name)), (item) =>
-      item.toUpperCase()
-    ).map((dataProvider, index) => ({
+    data.dataProviders.map((dataProvider, index) => ({
       id: index + 1,
-      value: dataProvider,
-      selected: dataProvider === queryVariables.dataProvider
+      value: dataProvider.name,
+      selected: dataProvider.name === queryVariables.dataProvider
```

Comment:
This line is to be sure that selected dataProvider on the app matches one of the listed dataProviders in the queryVariables ?

Reply:
this line sets `selected` to `true` or `false` depending on the `dataProvider.name` of the current step in the loop and the to that point selected data provider in query variables, which is the selected one from the dropdown.

**case 1:**
```
// current object in the loop
{
  "name": "OSF"
}

// queryVariables to that point
{
  "limit": 15,
  "dataProvider": "OSF"
}

// result from the map
{
  id: 1,
  value: "OSF",
  selected: true
}
```

**case 2:**
```
// current object in the loop
{
  "name": "rbb|24"
}

// queryVariables to that point
{
  "limit": 15,
  "dataProvider": "OSF"
}

// result from the map
{
  id: 2,
  value: "rbb|14",
  selected: false
}
```